apps/code-examples/src/app/code-examples/datetime/date-range-picker/help-key/demo.component.html

```diff
@@ -7,14 +7,6 @@
       [dateFormat]="dateFormat"
       [hintText]="hintText"
       [labelText]="labelText"
-      [required]="required"
-    >
-      @if (lastDonation.errors?.['dateWeekend']) {
```

Comment:
We're already demonstrating custom errors on the "basic" demo; I figured we should remove the extra bits to not distract from the "help key" functionality.